llvm/lib/ObjCopy/ELF/ELFObjcopy.cpp

```diff
@@ -623,6 +623,58 @@ handleUserSection(const NewSectionInfo &NewSection,
   return F(NewSection.SectionName, Data);
 }
 
+static Error verifyNoteSection(StringRef Name, llvm::endianness E,
```

Comment:
Nit: don't use single-letter variable names for things more complex than a loop counter. Consider `Endianness` instead.

Also, the `llvm::` qualifier is unnecessary. Same elsewhere.